srv/components/misc_pkgs/rabbitmq/files/rabbitmq-server

```diff
@@ -1,4 +1,5 @@
 /var/log/rabbitmq/*.log {
+    create 644 rabbitmq rabbitmq
     missingok
```

Comment:
@ypise 
su command was throwing below error and permissions was not getting set properly hence used create command.
error setting owner of /var/log/rabbitmq/rabbit@srvnode-1.log to uid 0 and gid 10: Operation not permitted

Reply:
@SomwanshiAnjali 
Looks better. This actually would apply to all files in the directory: `/var/log/rabbitmq/`